### PR TITLE
Cultivate relationship

### PR DESCRIPTION
**Pull request by chencuicru**

### CODE REVIEW

File: app/views/contacts/index.html.erb [hunk 1]
```diff
@@ -145,9 +145,7 @@
           </li>
           <li class="controls">
             <a href="#" ng-click="resetFilters()" class="btn">Reset Filters</a>
-            <% if $rollout.active?(:contacts_map, current_account_list) %>
```
Comment (twinge) [inline app/views/contacts/index.html.erb]: @soberstadt Did we activate the mapping feature for everyone already?

Comment (soberstadt) [inline app/views/contacts/index.html.erb]: oh yeah

Comment (twinge) [inline app/views/contacts/index.html.erb]: I just wanted to make sure it was ready because if it isn't, this commit will make it public

**Comment by soberstadt** [comment]
@chencuicru :+1: you're good to merge this.

**Comment by chencuicru** [comment]
*Merged*

On Thu, Jan 15, 2015 at 11:24 AM, Spencer Oberstadt <
notifications@github.com> wrote:

> @chencuicru <https://github.com/chencuicru> [image: :+1:] you're good to
> merge this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CruGlobal/mpdx/pull/240#issuecomment-70112626>.
>



-- 
*Chen Cui   *
Siebel Developer
Digital Strategies, Cru
407-826-2429
chen.cui@cru.org

**Pull request merged**